README.md

```diff
@@ -2,73 +2,51 @@
 
 [![Build](https://github.com/hpyproject/hpy/actions/workflows/ci.yml/badge.svg)](https://github.com/hpyproject/hpy/actions/workflows/ci.yml)
 [![Documentation](https://readthedocs.org/projects/hpy/badge/)](https://hpy.readthedocs.io/)
+[![Join the discord server at https://discord.gg/xSzxUbPkTQ](https://img.shields.io/discord/1077164940906995813.svg?color=7389D8&labelColor=6A7EC2&logo=discord&logoColor=ffffff&style=flat-square)](https://discord.gg/xSzxUbPkTQ)
 
-**IRC channel** : #hpy on [libera.chat](https://libera.chat)
-
+**Documentation**: [hpy.readthedocs.io](https://hpy.readthedocs.io/) \
```

Comment:
Maybe https://hpyproject.org or https://docs.hpyproject.org/en/latest/overview.html?

Reply:
Good point. Maybe it should be "Website: hpyproject.org" linking to https://hpyproject.org/?

Reply:
gr8